src/hotspot/cpu/riscv/matcher_riscv.hpp

```diff
@@ -129,7 +129,7 @@
 
   // Does the CPU supports vector variable shift instructions?
   static constexpr bool supports_vector_variable_shifts(void) {
-    return false;
+    return true;
```

Comment:
What's the path to checking for `UseZvbb` and `UseZvbc` respectively to the specific instruction?

Reply:
Here, the 3 checks are not independent, they depend on other checks in both vectorization and vector API, so returning true is fine.

But in order to eliminate everyone’s doubts in the future, I change them return UseZvbb.